Makefile

```diff
@@ -58,6 +58,24 @@ recrypt: import-gpg-keys
 			$(shell cat gpg.recipients | awk -F: {'printf "-r "$$1" "'}) && \
 		ansible-vault encrypt ${VAULT_FILE}
 
+test-aws:
```

Comment:
Makefile usage string on line 17 for the all: case needs updating to show the newly available test-aws and test-gce command options - A minor point, but nice to have it shown in the usage string.